meteor-app/imports/ui/assessment/assessment-job-card.js

```diff
@@ -14,27 +15,28 @@ class JobCard extends Component {
     const updatedStatus = status+1
 
     try {
-      if (status >= 5) return
-      Meteor.call('assessment.updateJobStatus', jobId, updatedStatus)
+      if (status >= JOB_STATUS.BIKE_PICKED_UP) return
+      this.props.updateStatus(jobId, updatedStatus)
     } catch (error) {
-      throw new Meteor.Error(error)
+      Alert.error(error)
     }
   }
 
   cancelButton = () => {
     const jobId = this.props.currentJob._id
     const status = this.props.currentJob.status
-    const cancelStatus = 6
-    const reopenStatus = 1
+    const cancelStatus = JOB_STATUS.CANCELLED
+    const reopenStatus = JOB_STATUS.NEW
+    const bikePickedUpStatus = JOB_STATUS.BIKE_PICKED_UP
     try {
-      if (status < 5) {
-        Meteor.call('assessment.updateJobStatus', jobId, cancelStatus)
-      } else if(status === 6) {
-        Meteor.call('assessment.updateJobStatus', jobId, reopenStatus)
+      if (status < bikePickedUpStatus) {
+        this.props.updateStatus(jobId, cancelStatus)
+      } else if(status === cancelStatus) {
+        this.props.updateStatus(jobId, reopenStatus)
       }
       return
     } catch (error) {
-      throw new Meteor.Error(error)
+      Alert.error(error)
```

Comment:
The `error` parameter is an object - I'm not sure if sAlert will handle this - you might need to pass `error.message` instead